deploy/README.md

```diff
@@ -88,9 +88,10 @@ The following table displays the tested Kubernetes and Helm versions.
 
 The following matrix displays the tested package versions for our Helm chart.
 
-| Sumo Logic Helm Chart | kube-prometheus-stack/Prometheus Operator | FluentD | Fluent Bit                          | Falco  | Metrics Server | Telegraf Operator | Tailing Sidecar Operator | OpenTelemetry Operator |
+| Sumo Logic Helm Chart | kube-prometheus-stack/Prometheus Operator | Fluentd | Fluent Bit                          | Falco  | Metrics Server | Telegraf Operator | Tailing Sidecar Operator | OpenTelemetry Operator |
```

Comment:
`Fluentd` or `FluentD`? :confused:

Reply:
https://www.fluentd.org/

![image](https://user-images.githubusercontent.com/70892616/210350307-5b1376fa-c108-4356-b820-57487b32e3e2.png)

Reply:
I feel like we are going back and forth :joy_cat: